src/test/java/rife/engine/TestEngine.java

```diff
@@ -542,13 +752,6 @@ void testGenerateEmptyFormPrefix()
         }
     }
 
-    class Routes extends Router {
```

Comment:
This class is located here because it's used by the test method that follows it directly.

Reply:
Your codestyle didn't think so.